test/test.js

```diff
@@ -646,6 +641,54 @@ test('assertions return promises', t => {
 	});
 });
 
+// https://github.com/avajs/ava/issues/1330
+test('no crash when adding assertions after the test has ended', t => {
+	t.plan(3);
+
+	ava(a => {
+		a.pass();
+		setTimeout(() => {
```

Comment:
I really don't think this should be ok. Many people do the mistake of using `setTimeout` inside a sync test and it silently passing. If users need to do this they should either use an async function or `test.cb()`.